### PR TITLE
fix(rest): Create new endpoint for LicenseType in admin tab

### PR DESCRIPTION
**Pull request by nikkuma7**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: #2146 

### How To Test?
1: Login with Admin user.
Sample URL : http://localhost:8080/resource/api/licenses/addLicenseType

> How should these changes be tested by the reviewer?
> Have you implemented any additional tests?

### Checklist
Must:
- [ ] All related issues are referenced in commit messages and in PR

### CODE REVIEW

File: rest/resource-server/src/docs/asciidoc/licenses.adoc [hunk 1]
```diff
@@ -142,3 +142,15 @@ include::{snippets}/should_document_import_osadl_info/curl-request.adoc[]
 
 ===== Example response
 include::{snippets}/should_document_import_osadl_info/http-response.adoc[]
+
+[[add-license-type]]
+==== create license type
```
Comment (rudra-superrr) [inline rest/resource-server/src/docs/asciidoc/licenses.adoc:147]: Change it to Create license type.

File: rest/resource-server/src/docs/asciidoc/licenses.adoc [hunk 2]
```diff
@@ -142,3 +142,15 @@ include::{snippets}/should_document_import_osadl_info/curl-request.adoc[]
...
+
+A `POST` request help to create license type.
+
```
Comment (rudra-superrr) [inline rest/resource-server/src/docs/asciidoc/licenses.adoc:150]: add request parameter for your POST request

File: rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/LicenseSpecTest.java [hunk 3]
```diff
@@ -255,12 +262,22 @@ public void should_document_upload_license() throws Exception {
...
+    @Test   		
     public void should_document_import_osadl_info() throws Exception {
         String accessToken = TestHelper.getAccessToken(mockMvc, testUserId, testUserPassword);
         mockMvc.perform(post("/api/licenses/import/OSADL")
                 .header("Authorization", "Bearer " + accessToken)
                 .accept(MediaTypes.HAL_JSON))
                 .andExpect(status().isOk());
     }
+
+	@Test
```
Comment (rudra-superrr) [inline rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/LicenseSpecTest.java:275]: use proper indentation

**Comment by rudra-superrr** [comment]
image

Testing failed.

**Comment by nikkuma7** [comment]
As per your above comment, code has modified. Please retest.

**Comment by rudra-superrr** [comment]
Testing was successful.

File: rest/resource-server/src/docs/asciidoc/licenses.adoc [hunk 4]
```diff
@@ -126,19 +126,57 @@ include::{snippets}/should_document_get_download_license_archive/http-response.a
 
 A `POST` request will upload license file.
 
+[black]#Request parameter#
```
Comment (rudra-superrr) [inline rest/resource-server/src/docs/asciidoc/licenses.adoc:129]: don't style the headings with black.

Comment (nikkuma7) [reply rest/resource-server/src/docs/asciidoc/licenses.adoc]: done.

**Comment by rudra-superrr** [comment]
Code looks good.

**Pull request merged**